viewer/packages/pointclouds/src/CognitePointCloudModel.ts

```diff
@@ -212,16 +216,91 @@ export class CognitePointCloudModel extends THREE.Object3D implements CogniteMod
   }
 
   /**
-   * Gets the stylable objects associated with this point cloud.
+   * Gets default point appearance
+   */
+  getDefaultPointCloudAppearance(): PointCloudAppearance {
+    return this.pointCloudNode.defaultAppearance;
+  }
+
+  /**
+   * Sets default apparance for points that are
+   * not styled otherwise
+   */
+  setDefaultPointCloudAppearance(appearance: Partial<PointCloudAppearance>): void {
+    const fullAppearance: PointCloudAppearance = applyDefaultsToPointCloudAppearance(appearance);
+    this.pointCloudNode.defaultAppearance = fullAppearance;
+  }
+
+  /**
+   * Gets the object collections that have been assigned a style
+   */
+  get styledCollections(): StyledPointCloudObjectCollection[] {
+    return this._styledObjectCollections;
+  }
+
+  /**
+   * Assign a style to a collection of objects
+   */
+  assignStyledObjectCollection(
+    objectCollection: PointCloudObjectCollection,
+    appearance: Partial<PointCloudAppearance>
```

Comment:
This API differs a bit from corresponding CAD API. I think they should be the same, i.e. either `PointCloudAppearance` members becoming optional or `NodeAppearance` members becoming required and use Partial there too (make sure API docs works well if we use Partial)

Reply:
Made it non-partial, but introduced a `CompletePointCloudObjectAppearance`- type that I use internally instead.